community/modules/compute/schedmd-slurm-gcp-v6-partition/main.tf

```diff
@@ -19,12 +19,13 @@ locals {
   has_node = length(var.nodeset) > 0
   has_dyn  = length(var.nodeset_dyn) > 0
   has_tpu  = length(var.nodeset_tpu) > 0
+  has_flex = length([for ns in var.nodeset : ns.dws_flex.enabled if ns.dws_flex.enabled]) > 0 ? true : false
```

Comment:
This ternary expression seems to be unnecessary.
will simply `length([for ns in var.nodeset : ns.dws_flex.enabled if ns.dws_flex.enabled]) > 0` be sufficient?

Reply:
Good point, not sure why I did it for `use_job_duration` above...removed both instances.